tests/bluetooth/iso/bsim/tests/2RTN_1000ms_ACL.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+SCRIPT_NAME=$(basename "$0")
+
+${SCRIPT_DIR}/../iso_and_acl.sh ${SCRIPT_NAME} \
+  "iso_brcast_src" "rtn_set" "2" \
+  "acl_central" "send_int_set" "1000" \
+  "acl_central" "payload_size_set" "20" \
+  "tester" "acl_fail_limit_percent" "0" \
+  "tester" "iso_fail_limit_percent" "1" \
+  "tester" "acl_packets_limit_min" "3" \
+  "tester" "iso_packets_limit_min" "800"
```

Comment:
To demonstrate my earlier comment about the arguments, this could have been

```suggestion
  "tester" "acl" "fail_limit_percent" "0" "packets_limit_min" "3" \
  "tester" "iso" "fail_limit_percent" "1" "packets_limit_min" "800" \
```